addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -890,13 +890,13 @@ def _l10n_it_post_invoices_step_2(self, invoices):
         to_return = {}
         company = invoices.company_id
 
-        proxy_user = self._get_proxy_user(company)
+        proxy_user = self.env['account_edi_proxy_client.user']._get_proxy_users(company, 'l10n_it_edi')
```

Comment:
at the moment we don't allow creating a separate user per edi mode, so this one can only return one edi user.